tests/RobotFramework/tests/cumulocity/configuration/legacy_config_operation/legacy_configuration_workflow.robot

```diff
@@ -0,0 +1,47 @@
+*** Settings ***
+Resource            ../../../../resources/common.resource
+Library             Cumulocity
+Library             ThinEdgeIO
+
+Test Setup          Custom Setup
+Test Teardown       Get Logs
+
+Test Tags           theme:c8y    theme:configuration
+
+
+*** Test Cases ***
+Support Legacy Configuration Download Operation
+    [Documentation]    Support Cumulocity Legacy File Upload/Download
+    ...    which was the default before file-type configuration operations were introduced
+    ...    Cumulocity Docs: https://cumulocity.com/docs/device-integration/fragment-library/#install-legacy-configuration
+
+    # pre-condition
+    Cumulocity.Managed Object Should Not Have Fragments    c8y_ConfigurationDump
+
+    # Send configuration to a device
+    ${binary_url}=    Create Inventory Binary    dummy    example    contents=foobar
+    ${operation}=    Cumulocity.Create Operation
+    ...    description=Legacy Configuration Download
+    ...    fragments={"c8y_DownloadConfigFile":{"url": "${binary_url}", "c8y_ConfigurationDump": {"id": "1111"}}}
```

Comment:
This `id` should be the binary ID to render the content in the UI.
For example, if `${binary_url}` is `https://rina.cumulocity.com/inventory/binaries/46206`, it should be `46206`.

Reply:
Actually I actively chose a different value for the c8y_ConfigurationDump.id as I was curious to see if it was used in the update or if the url was somehow parsed by the server.

Reply:
Technically the `c8y_ConfigurationDump.id` an be different to the `.url` field, so having them use different values ensures that we've validating where the inventory managed object fragment is coming from.

Reply:
After our conversation, I've updated the system test to use the binary id taken from the binary's URL (to replicate what is done in the Cumulocity UI).

Commit 64ddcfb2160848ee7acbb9e1768ddea3efe8d7c3